docs/source/user/debugger.rst

```diff
@@ -28,7 +28,7 @@ Here is an example of how to install ``ipykernel`` and ``xeus-python`` in a new
 
 .. code:: bash
 
-   conda create -n jupyterlab-debugger -c conda-forge jupyterlab=3 ipykernel>=6 xeus-python
+   conda create -n jupyterlab-debugger -c conda-forge jupyterlab=3 'ipykernel>=6' xeus-python
```

Comment:
Thanks for catching this one

let's use double quote (for those using Windows CMD shell):

```suggestion
   conda create -n jupyterlab-debugger -c conda-forge jupyterlab=3 "ipykernel>=6" xeus-python
```

Reply:
Ah right, forgot about that difference on Windows! Committed.